content/about.html

```diff
@@ -18,24 +18,23 @@
           </div>
           <div class="masthead-left">
             <h1 class="masthead-left__title no-link">
-              What is web.dev?
+              Where you can learn & measure
             </h1>
             <div class="masthead-left__body">
               <p>
-                web.dev is the ultimate resource for developers of all
-                backgrounds to learn, create, and solve on the web. It's meant
-                to not only educate developers, but help them apply what they've
-                learned to any site they work on, be it personal or business.
+                Whether you already have a website or you're just getting started, you 
```

Comment:
Suggestion to remove "you can" to take a line off a paragraph. So it's "Whether you already have a website or you're just getting started, learn to build for the modern web at web.dev."